### PR TITLE
Created Hello World using aime And Added Hyperlink for every File in README.md

### LINKED ISSUE #39: Readme: Link from list to example
Every language should linked to the hello-world example file.

### PR DESCRIPTION
**Pull request by Roshians**

### CODE REVIEW

File: aime.aime [hunk 1]
```diff
@@ -0,0 +1,7 @@
+integer
+main(void)
+{
+    o_text("Hello World\n");
+ 
+    return 0;
+}
```
Comment (Den4200) [inline aime.aime:7]: From my findings, it seems like you'd only need this:
```aime
o_text("Hello world!\n");
```
Is there any reason to add the extra code in that you have?

**Review by Den4200 — CHANGES_REQUESTED** [review]
Thanks so much for adding links to each implementation! I only have one small question.

File: aime.aime (hunk 1 above)
Comment (Roshians) [reply aime.aime:7]: Yes, this can also be written. And produce the same output

Comment (Den4200) [reply aime.aime:7]: Alright, I can't particularly test this, so I'm trusting you here.

**Review by Den4200 — APPROVED** [review]
Awesome, thanks again!

**Pull request merged**